test/test_PublicInterfaceImpl.c

```diff
@@ -65,165 +69,793 @@ void setUp(void)
 
     BSL_SecParam_t param_scope_flag = { 0 };
     BSL_SecParam_InitInt64(&param_scope_flag, RFC9173_BIB_PARAMID_INTEG_SCOPE_FLAG, 0);
+    BSL_SecParam_t param_scope_flag_7 = { 0 };
```

Comment:
Again, not a problem introduced here, but if a struct does have an `_Init()` function defined, then using an initializer list is duplicate / wasted because the Init func should do the right thing on an uninitialized starting state.